File: 가장 큰 수 찾기.java
class Solution {
    public int[] solution(int[] array) {
        int max=0;
        int n =0;
        for(int i =0;i<array.length;i++){
            if(array[i]>max){
                max=array[i];
                n=i;
            }
        }
        int[] answer = {max,n};
        return answer;
    }
}

File: 3st/Juram/programmer/가장 큰 수 찾기.js
function solution(array) {
    var answer = [];
    let max;
    let idx;
    for(let i=0; i<array.length-1; i++){
        for(let j=1; j<array.length; j++){
            if(array[i]<array[j]){
                max = array[j];
                idx = j;
            }
            else{
                max = array[i];
                idx = i;
            }
        }
    }
    answer.push(max);
    answer.push(idx);
    
    return answer;
}